### ISSUE TITLE
Q: How to avoid some list positions from swiping ?

### ISSUE THREAD

**Issue by marioosh-net** [47deg/android-swipelistview#244]
Any ideas how to do that ?

**Comment by PerfactLiu** [comment]
you can set swipeMode

**Comment by marioosh-net** [comment]
Yes, i can `setSwipeMode` to none, but for `SwipeListView` at all, but need avoid only **selected** positions from responding on swipe.

**Comment by PerfactLiu** [comment]
last week tell you

**Comment by PerfactLiu** [comment]
next week ,i have to go

**Comment by Fenil15** [comment]
@marioosh-net did you able to figure it out?

**Comment by mateusgrb** [comment]
Use Item View Types. Override getItemViewType in your adapter and then custom SwipeListView to only allow swiping for certain view types.

http://developer.android.com/reference/android/widget/Adapter.html#getItemViewType(int)

**Comment by Fenil15** [comment]
Easier way is to override isEnabled property of Adapter, setting it to false for required item position will disable the swipe functionality
